Backend/src/modules/users/users.controller.ts

```diff
@@ -42,4 +43,44 @@ export default class UserController extends Api {
       );
     }
   };
+
+  public UpdateUserInfo = async (
+    req: Request<unknown, unknown, UpdateUserRequestDto>,
+    res: Response
+  ) => {
+    try {
+      // Check if req.file is defined before accessing its filename property
+      const imageFilePath: string | undefined = req.file?.filename
+        ? encodeURIComponent(req.file.filename)
+        : undefined;
+      if (!imageFilePath) {
+        throw new HttpBadRequestError('No image provided');
+      }
```

Comment:
User might change his name only. not his profile. and might change birthdate/age only. So we need to be able to handle partial update